assets/css/_sass/website/_shame.scss

```diff
@@ -68,3 +72,69 @@ p {
     line-height: 1.5em;
   }
 }
+
+
+.avatar-list {
```

Comment:
I think we can move these items specifically for the contact us page to their own file rather than living in the `_shame.scss` file. It's not really a shame category to me. Up to you, though.